ddm-admin-client/build.rs

```diff
@@ -0,0 +1,99 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// Copyright 2022 Oxide Computer Company
+
+use anyhow::bail;
+use anyhow::Context;
+use anyhow::Result;
+use quote::quote;
+use serde::Deserialize;
+use std::collections::BTreeMap;
+use std::env;
+use std::fs;
+use std::path::Path;
+
+// TODO-cleanup This is a copy/paste of (a subset of) omicron_package::Config.
+// We ought to depend on it directly, but that creates a circular dependency
+// between crates:
+//
+// us -> omicron_package -> omicron_sled_agent -> us
+//
+// omicron_package only depends on sled-agent for zone management; maybe that
```

Comment:
We have actually been pushing out the packaging tools from the "omicron" repo to https://github.com/oxidecomputer/omicron-package , so that they can be used by other crates too.

Anything in Omicron should be able to depend on that crate (which actually happens to be named `omicron-zone-package`), if you want to upgrade the structures to exist there.

Reply:
Fixed in eb59617d (via https://github.com/oxidecomputer/omicron-package/pull/15, which @smklein  just published - thanks!).